uber/templates/macros.html

```diff
@@ -162,7 +162,7 @@
         });
     </script>
 
-    {% if is_prereg_dealer %}
+    {% if is_dealer and not group %}
```

Comment:
Wait, this was a good change that you made... we should check c.COLLECT_FULL_ADDRESS here because the JS is useless if the full personal address isn't collected.

I don't think we need it passed to the macro cause it's a global var, but still!